### PR TITLE
Qt script

### PR DESCRIPTION
**Pull request by j8xixo12**
add qt build script and modified netcdf md5 value

### CODE REVIEW

File: scripts/build.d/qt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+       echo "perl not exist!"
+       exit 1
+    fi
+    perl init-repository -f
+  fi
+
+  mkdir -p build
+  cd build
+  cmakecmd=("cmake")
+  cmakecmd+=("-DCMAKE_INSTALL_PREFIX=${DEVENVPREFIX}")
```
Comment (yungyuc) [inline scripts/build.d/qt:43]: I'd guess it need more arguments for proper configuration.  Perhaps [the Qt3D issue](https://github.com/solvcon/modmesh/issues/171) has something to do with it.  We can investigate later.

**Review by yungyuc — APPROVED** [review]
LGTM.  Thanks.

File: scripts/build.d/qt [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  mkdir -p ${DEVENVFLAVORROOT}/${DEVENVFLAVOR}/src
+  pushd ${DEVENVFLAVORROOT}/${DEVENVFLAVOR}/src > /dev/null
+    unzip ${DEVENVDLROOT}/$pkgfn
+    mv $pkgfoler $pkgfull
+  popd
+else
+  pkgname=qt
+  pkgbranch=${VERSION:-6.4.1}
+  pkgfull=$pkgname-$pkgbranch
+  syncgit git://code.qt.io/qt qt5 ${pkgbranch} ${pkgfull}
```
Comment (tai271828) [inline scripts/build.d/qt:28]: Is `qt5` expected?

**Review by tai271828 — APPROVED** [review]
I would say it looks good to me. One minor comment inline for your reference.

File: scripts/build.d/qt (hunk 2 above)
Comment (j8xixo12) [reply scripts/build.d/qt:28]: Yes, qt5 is expected!
[official doc](https://wiki.qt.io/Building_Qt_6_from_Git#Getting_the_source_code)

Comment (yungyuc) [reply scripts/build.d/qt:28]: Oh, it looks so much like a bug.  @j8xixo12 perhaps you can add the link in the code comment to make it obvious for maintainers.

**Review by chestercheng — APPROVED** [review]

File: scripts/build.d/qt (hunk 2 above)
Comment (tai271828) [reply scripts/build.d/qt:28]: + 1 for adding comments, and then we can land the code.

Comment (j8xixo12) [reply scripts/build.d/qt:30]: @yungyuc @tai271828 I just added comments for qt repo name.

**Pull request merged**